logstash-core/lib/logstash/config/source_loader.rb

```diff
@@ -0,0 +1,68 @@
+# encoding: utf-8
+require "logstash/config/source/local"
+require "logstash/errors"
+require "thread"
+require "set"
+
+module LogStash module Config
+  class SourceLoader
+    class Loader
+      def initialize(loaders)
```

Comment:
I think naming isn't very consistent in this namespace, I'll point out some of these observations below